app/Services/Manager.php

```diff
@@ -495,4 +495,27 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     */
+    public function sendEmail($user, $contest, $params)
+    {
+        $message = Message::where($params)->first();
```

Comment:
hmm i feel like that would then cause the query to just return all messages. I feel like requiring it is important and allows the developer to easily see where something failed. I did add the check to see if a message was found our not with a log message that says what params were passed, which could be useful.